phpcs.xml.dist

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards for AMP Theme">
+
+	<rule ref="WordPress-Core" />
+	<rule ref="WordPress-Docs" />
+	<rule ref="WordPress-Extra" />
+
+	<rule ref="WordPress.WP.I18n">
+		<properties>
+			<property name="text_domain" value="amp" />
```

Comment:
Hi @ThierryA,
The `amp-wp` plugin also [has the textdomain](https://github.com/Automattic/amp-wp/blob/0.6/amp.php#L96) of `amp`. Should this theme have the same textdomain as that?

Reply:
Thanks @kienstra, I created a separate issue for that [here](https://github.com/xwp/wp-amp-theme/issues/5).